modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -4,147 +4,58 @@
-|
-|{product-title} automatically generates the `UID` and completes the `namespace` with the name of the project where the object is created.
-
-|`name:`
-|Name for the object.
-|Name of your TempoStack instance.
-|`tempo-all-in-one-inmemory`
-
-|`spec:`
-|Specification for the object to be created.
-|Contains all of the configuration parameters for your TempoStack instance. When a common definition for all Tempo components is required, it is defined under the `spec` node. When the definition relates to an individual component, it is placed under the `spec/template/<component>` node.
-|N/A
-
-|`resources:`
-|Resources assigned to the TempoStack instance.
-|
-|
-
-|`storageSize:`
-|Storage size for ingester PVCs.
-|
-|
-
-|`replicationFactor:`
-|Configuration for the replication factor.
-|
-|
-
-|`retention:`
-|Configuration options for retention of traces.
-|
-|
-
-|`storage:`
-|Configuration options that define the storage. All storage-related options must be placed under `storage` and not under the `allInOne` or other component options.
-|
-|
-
-|`template.distributor:`
-|Configuration options for the Tempo `distributor`.
-|
-|
-
-|`template.ingester:`
-|Configuration options for the Tempo `ingester`.
-|
-|
-
-|`template.compactor:`
-|Configuration options for the Tempo `compactor`.
-|
-|
-
-|`template.querier:`
-|Configuration options for the Tempo `querier`.
-|
-|
-
-|`template.queryFrontend:`
-|Configuration options for the Tempo `query-frontend`.
-|
-|
-
-|`template.gateway:`
-|Configuration options for the Tempo `gateway`.
-|
-|
-
-|===
-
-
-
-.Minimum required configuration
-
-The following is the required minimum for creating a {TempoShortName} deployment with the default settings:
-
-[source,yaml]
-----
-apiVersion: tempo.grafana.com/v1alpha1
-kind: TempoStack
-metadata:
-  name: simplest
-spec:
-  storage: # <1>
-    secret:
-      name: minio
-      type: s3
-  resources:
-    total:
-      limits:
-        memory: 2Gi
-        cpu: 2000m
-  template:
-    queryFrontend:
-      jaegerQuery:
-        enabled: true
-        ingress:
-          type: route
-----
-<1> This section specifies the deployed object storage back end, which requires a created secret with credentials for access to the object storage.
+<1> API version to use when creating the object.
+<2> Defines the kind of Kubernetes object to create.
+<3> Data that uniquely identifies the object, including a `name` string, `UID`, and optional `namespace`. {product-title} automatically generates the `UID` and completes the `namespace` with the name of the project where the object is created.
+<4> Name of the TempoStack instance.
+<5> Contains all of the configuration parameters of the TempoStack instance. When a common definition for all Tempo components is required, it is defined under the `spec` node. When the definition relates to an individual component, it is placed under the `spec/template/<component>` node.
```

Comment:
```suggestion
<5> Contains all of the configuration parameters of the `TempoStack` instance. When a common definition for all Tempo components is required, it is defined under the `spec` node. When the definition relates to an individual component, it is placed under the `spec/template/<component>` node.
```
`TempoStack` should always be formatted using backticks since it's a specific API object. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

Reply:
Also, I wouldn't say "node" here, since nodes mean something specific in Kubernetes / OCP. I would say maybe "`spec` field" or "`spec` section"

Reply:
I would also use the format `spec.template.<component>` rather than `spec/template/<component>`, since this is the generally used annotation for referring to Kubernetes / OCP API fields.

Reply:
Thank you so much. I've updated this line.